src/test/webapp/WEB-INF/spring/hazelcast-spring.xml

```diff
@@ -42,10 +42,6 @@
             <param-name>shutdown-on-destroy</param-name>
             <param-value>true</param-value>
         </init-param>
-        <init-param>
-            <param-name>client-config-location</param-name>
```

Comment:
why are those deleted in all configurations? `client-config-location` and `config-location`